### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by lbwexler**
Hoist P/R Checklist
-------------------

**Pull request authors:** Review and check off the below. Items that do not apply can also be
checked off to indicate they have been considered. If unclear if a step is relevant, please leave
unchecked and note in comments.

- [X] Caught up with `develop` branch as of last change.
- [X] Added CHANGELOG entry, or determined not required.
- [X] Reviewed for breaking changes, added `breaking-change` label + CHANGELOG if so.
- [X] Updated doc comments / prop-types, or determined not required.
- [X] Reviewed and tested on Mobile, or determined not required.
- [X] Created Toolbox branch / PR, or determined not required.

**If your change is still a WIP**, please use the "Create draft pull request" option in the split
button below to indicate it is not ready yet for a final review.

> **Pull request reviewers:** when merging this P/R, please consider using a *squash commit* to
> collapse multiple intermediate commits into a single commit representing the overall feature
> change. This helps keep the commit log clean and easy to scan across releases. PRs containing a
> single commit should be *rebased* when possible.

### CODE REVIEW

File: admin/tabs/general/alertBanner/AlertBannerModel.ts [hunk 1]
```diff
@@ -69,7 +68,7 @@ export class AlertBannerModel extends HoistModel {
     }
 
     get supportPresets() {
-        return checkMinVersion(XH.getEnv('hoistCoreVersion'), '16.3.0');
+        return XH.environmentService.isMinHoistCoreVersion('16.3.0');
```
Comment (amcclain) [inline admin/tabs/general/alertBanner/AlertBannerModel.ts:71]: Woohoo - saved us two chars! 😉 

We could alias off of `XH` if we wanted - I appreciate the clarity of the long method name, though

File: admin/tabs/server/logViewer/LogViewerModel.ts [hunk 2]
```diff
@@ -51,7 +50,6 @@ export class LogViewerModel extends HoistModel {
         text: 'Download',
         icon: Icon.download(),
         recordsRequired: 1,
-        disabled: !checkMinVersion(XH.environmentService.get('hoistCoreVersion'), '9.4'),
```
Comment (amcclain) [inline admin/tabs/server/logViewer/LogViewerModel.ts:54]: Wow!

**Review by amcclain — COMMENTED** [review]
Great cleanups! ✨

**Comment by lbwexler** [comment]
yea, thought of doing the alias as well -- was on the fence -- can do that
at any point.   By the way,  did try a function from utils, but ran into
circular dependency issues.... with trying to import XH from the lowest
level dependency....

**Pull request merged**